package.json

```diff
@@ -9,8 +9,9 @@
     "bundle": "wireit",
     "lint": "wireit",
     "lint:fix": "wireit",
-    "lint:cpp": "find src packages/realm/bindgen/src -name '*.cpp' -or -name '*.h' -or -name '*.hpp' | xargs clang-format --dry-run --Werror",
-    "lint:cpp:fix": "find src packages/realm/bindgen/src -name '*.cpp' -or -name '*.h' -or -name '*.hpp' | xargs clang-format -i",
+    "trunk:check": "trunk check",
```

Comment:
Any reason for not calling it `lint:cpp`?

Reply:
I think it might make more sense to keep the name more generic, like the old one Kenneth is referring to here, to not have the name tied to the underlying tool. Also, since `trunk` is not super obvious what it does 😉  (And if so, similarly for the `name` in the [workflow](https://github.com/realm/realm-js/blob/bb64dd0ffe1d6ed452203f09c2227ef7f331a9b8/.github/workflows/pr-linting.yml#L23).)

Reply:
It would encode an implicit expectation that the `.trunk/trunk.yaml` only enables linters for "cpp" files, introducing a coupling between this name and the configuration file, which I'd like to avoid.

But I agree `trunk` in itself doesn't make too much sense, how about `lint:trunk`? (lint being the semantic thing and trunk the tool - or perhaps I should just move this into the "command" for the existing "lint" script 🤔

Reply:
Moving it into the existing script seems like the better option then 👍